scripts/cassandra-integration-test.sh

```diff
@@ -37,6 +37,7 @@ apply_schema() {
     --env CQLSH_PORT=9042
     --env "TEMPLATE=/cassandra-schema/${schema_version}.cql.tmpl"
     --env "KEYSPACE=${keyspace}"
+    --env "DB_PASSWORD=${DB_PASSWORD}"
```

Comment:
- why only password and not user name?
- we don't need any env variables, the pwd can be simply hardcoded here, since the db container is created just for this test

Reply:
@yurishkuro Then what is the password?

Reply:
whatever, make up a string, e.g. `testpwd`